prisma/schema.prisma

```diff
@@ -9,3 +9,40 @@ datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
+
+model User {
+  id            Int      @id @default(autoincrement())
+  email         String   @unique
+  dateCreated   DateTime @default(now())
+  username      String   @unique @db.VarChar(60)
+  password      String   @db.VarChar(60)
+  writtenZoops  Zoop[]   @relation("Author")
+  receivedZoops Zoop[]   @relation("Receiver")
```

Comment:
Same. Would be easier for glancing at it. But this is a style but, your call if you want to bother fixing and trying to run another migration